projects/storefrontstyles/scss/_versioning.scss

```diff
@@ -64,6 +64,81 @@ $useLatestStyles: false !default;
   }
 }
 
+@function getVersion($value) {
+  $major: 0;
+  $minor: 0;
+
+  $string: toString($value);
+  $i: str-index($string, '.');
+  @if $i != null {
+    $major: toNumber(str-slice($string, 1, $i - 1));
+    $minor: toNumber(str-slice($string, $i + 1));
+  } @else {
+    $major: toNumber($string);
+  }
+
+  @return (major: $major, minor: $minor);
+}
+
+@function toString($value) {
+  @if type-of($value) == 'number' {
+    @return #{$value};
+  } @else if type-of($value) == 'string' {
+    @return $value;
+  } @else if type-of($value) == 'list' {
```

Comment:
In which case would there be a `list`?

Reply:
`1.2.3` will be `[1.2, 0.3]`; `1.2.3.4` will be `[1.2, 0.3, 0.4]`. So, customer can set any dot numbers, but we only get the first item in list `toString(nth($value, 1));`, that is `1.2`.